systemd/systemd.go

```diff
@@ -412,16 +412,21 @@ func MountUnitPath(baseDir, ext string) string {
 }
 
 func (s *systemd) WriteMountUnitFile(name, what, where string) (string, error) {
+	extra := ""
+	if osutil.IsDirectory(what) {
+		extra = "Options=bind\nType=none\n"
+	}
+
```

Comment:
I suppose this is fairly generic behavior so it makes sense here, should we follow the symlink before getting here or not, mmh?